### ISSUE TITLE
Beware feature importances

### ISSUE THREAD

**Issue by mattc-eostar** [WillKoehrsen/feature-selector#43]
https://explained.ai/rf-importance/

Are you aware of this method and its benefits?

https://scikit-learn.org/stable/modules/generated/sklearn.inspection.permutation_importance.html?highlight=permutation#sklearn.inspection.permutation_importance

Might be a slightly situation using gradient boosted trees, but still an interesting point to address if you could. Thanks!

**Comment by BrandonKMLee** [comment]
I think there are also other forms of importance, but they may or may not be useful.
https://github.com/interpretml/interpret/issues/369
